code/game/gamemodes/changeling/powers/revive.dm

```diff
@@ -6,9 +6,9 @@
 
 //Revive from regenerative stasis
 /obj/effect/proc_holder/changeling/revive/sting_action(var/mob/living/carbon/user)
-	user.setToxLoss(0)
-	user.setOxyLoss(0)
-	user.setCloneLoss(0)
+	user.setToxLoss(0, FALSE)
+	user.setOxyLoss(0, FALSE)
+	user.setCloneLoss(0, FALSE)
 	user.setBrainLoss(0)
 	user.SetParalysis(0)
```

Comment:
shouoldn't stun/paralysis/weakened calls also not update since `updatehealth` is called at the end?

Reply:
Ah, that is true